### PR TITLE
Describe the Form - Customize Items demo

### PR DESCRIPTION
**Pull request by AlexJameson**

### CODE REVIEW

**Review by RomanTsukanov — DISMISSED** [review]

File: JSDemos/Demos/Form/CustomizeItem/description.md [hunk 1]
```diff
@@ -1 +1,5 @@
-The Form component can create form items automatically for each field of&nbsp;the **formData** object. If&nbsp;you&rsquo;d prefer to&nbsp;configure form items manually, pass the required array of&nbsp;configuration objects to&nbsp;the **items** property. You can specify the name of&nbsp;the bound data field, editor type, editor configuration, validation properties (etc.) for each form item.
\ No newline at end of file
+You can use the [items[]](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#items) array to configure all form items. This array can contain strings ([formData](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#formData) field names) and objects (item configurations). Use a string to create a simple item with default configuration as shown for `Email` item.
```
Comment (Abadzhev) [inline JSDemos/Demos/Form/CustomizeItem/description.md]: I would move the last sentence to a separate paragraph. That way you'll have three paragraphs / "pieces of information":

- Paragraph 1: what's the entry point for configuration, what are the two types. 
- Paragraph 2: first type (simple string)
- Paragraph 3: second type (objects) 

Right now you describe two types, then list the first, and then the thought breaks off.

File: JSDemos/Demos/Form/CustomizeItem/description.md [hunk 2]
```diff
@@ -1 +1,5 @@
...
+
+To change the default settings, declare an item configuration object. Use the [dataField](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#dataField) property to bind an item to a field in the **formData** object. Use the [editorType](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#editorType) property to specify an item's data editor or configure the editor in the [editorOptions](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#editorOptions) object. You can also specify any other properties described in the [SimpleItem](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/) section.
+
+This demo shows how to specify **editorOptions**, **editorType**, [validationRules](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#validationRules), and [colSpan](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#colSpan) properties for the Form simple items.
```
Comment (Abadzhev) [inline JSDemos/Demos/Form/CustomizeItem/description.md]: The article with complicated noun phrase in plural doesn't read well to me. Both possessives and articles with plural are tricky and sometimes it's better to break a noun phrase apart and introduce a preposition (even "of"), just to be clear. 

```suggestion
This demo shows how to specify **editorOptions**, **editorType**, [validationRules](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#validationRules), and [colSpan](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#colSpan) properties for simple items in a Form widget.
```

File: JSDemos/Demos/Form/CustomizeItem/description.md (hunk 1 above)
Comment (dirk-pieterse) [inline JSDemos/Demos/Form/CustomizeItem/description.md]:
```suggestion
You can use the [items[]](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#items) array to configure all form items. This array can contain strings ([formData](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#formData) field names) and objects (item configurations). Use a string to create a simple item with default configuration as shown for the `Email` item. 
```

**Review by dirk-pieterse — APPROVED** [review]

**Review by dirk-pieterse — DISMISSED** [review]

File: JSDemos/Demos/Form/CustomizeItem/description.md [hunk 3]
```diff
@@ -1 +1,5 @@
...
+
+To change the default settings, declare an item configuration object. Use the [dataField](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#dataField) property to bind an item to a field in the **formData** object. Use the [editorType](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#editorType) property to specify an item's data editor or configure the editor in the [editorOptions](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#editorOptions) object. You can also specify any other properties described in the [SimpleItem](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/) section.
+
+This demo shows how to specify **editorOptions**, **editorType**, [validationRules](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#validationRules), and [colSpan](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#colSpan) properties for simple items in a Form widget.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/Form/CustomizeItem/description.md]:
```suggestion
This demo shows how to specify **editorOptions**, **editorType**, [validationRules](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#validationRules), and [colSpan](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#colSpan) properties for simple items in a Form component.
```

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**